### DISCUSSION TITLE
ESP32

### DISCUSSION THREAD

**Discussion by brendan1495** [pmgducati/Blue-Retro-AIO-Units#77] [Q&A]
Would the "ESP32-DevKit C development board WIFI Bluetooth development board 38Pin TYPE-C interface equipped with ESP32-D0WD chip" work with this I have already bought a couple the first was to narrow and the second not enough pins, Any help would be appreciated thanks

**Comment by pmgducati** [comment]
Can you post pictures of the boards that you have?

**Comment by brendan1495** [comment]
Hi Pictures attached 

![IMG_20240320_110208_391](https://github.com/pmgducati/Blue-Retro-AIO-Units/assets/8987747/0e1edbaf-e297-40f5-8141-7b31a9ae4fd2)


![IMG_20240320_110225_382](https://github.com/pmgducati/Blue-Retro-AIO-Units/assets/8987747/88f6e1e7-fb9a-4e03-aa61-5d24ea812fb6)

![IMG_20240320_110208_391](https://github.com/pmgducati/Blue-Retro-AIO-Units/assets/8987747/686cbcf7-e9b6-4127-8201-249247f98a69)
![IMG_20240320_110225_382](https://github.com/pmgducati/Blue-Retro-AIO-Units/assets/8987747/8eef8d7c-d3c7-4b01-b926-298de7e13d2c)

the ones I am looking at are:

https://www.aliexpress.com/item/1005006500102606.html?spm=a2g0o.productlist.main.59.51eb2d99vFdfBK&algo_pvid=85f10390-810b-4420-b4fe-0e9238e522af&algo_exp_id=85f10390-810b-4420-b4fe-0e9238e522af-29&pdp_npi=4%40dis%21GBP%2110.89%213.05%21%21%2196.92%2127.14%21%402101f93417108898928833138e58d2%2112000037428660369%21sea%21UK%212052184844%21&curPageLogUid=Bmon541Y6DlX&utparam-url=scene%3Asearch%7Cquery_from%3A

Thanks, I have probably uploaded the pictures twice

**Comment by brendan1495** [comment]
Any ideas?
